_posts/2020-10-03-hacktoberfest.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "Hacktoberfest: let's start to contribute to Strimzi"
+date: 2020-10-03
+author: ppatierno
+---
+
+Finally it is October and, as every year, it is the time of the [Hacktoberfest](https://hacktoberfest.digitalocean.com/)!
+As stated on the official website, "Hacktoberfest is a monthlong celebration of open source software" and it is open to everyone for contributing to open source projects.
+So ... why not contributing to Strimzi and getting some swags as well?
+
+
+<!-- more -->
+
+![Overview](/assets/images/posts/2020-10-03-hacktoberfest.png)
+
+Run by DigitalOcean, the 2020 is the 7th edition of this event for promoting people all around the world to contribute to open source projects.
+Whether you are a developer, a student, a techical writer, a company, you can help communities to improve the projects they develop and host on GitHub.
+It's done as a challenge: you have to contribute with at least four pull requests between October 1st and October 31st in order to get a limited edition of the celebrating T-shirt or plant a tree.
+You can find more information about the rules and the process to onboarding, on the official [website](https://hacktoberfest.digitalocean.com/).
+
+Maybe you are asking to yourself, "why talking about Hacktoberfest on the Strimzi website?"
+It is simple! Right?
+
+The Strimzi community would be really happy to take part at this event and having more people to get involved with contributions.
+After the last blog post, about the first [Strimzi survey results](https://strimzi.io/blog/2020/09/29/strimzi-survey-results/), we noticed that developers would love to contribute to Strimzi and they have planned to do so.
+So, what is stopping you?
+What could be a better opportunity than starting with the Hacktoberfest?
+
+Frome simpler to more complex ones, from code to documentation, you can find different [issues](https://github.com/strimzi/strimzi-kafka-operator/issues) on our GitHub repository in order to engage with us and starting to contribute.
+If you don't find something that can fit your knowledge or you have any kind of problems and need advices, don't hesitate to ask us anything in the Slack room or on mailing list.
```

Comment:
I think you should go into more details here and point directly to some of the labels:
* https://github.com/strimzi/strimzi-kafka-operator/labels/good-start
* https://github.com/strimzi/strimzi-kafka-operator/labels/documentation
* maybe https://github.com/strimzi/strimzi-kafka-operator/labels/help%20wanted

That will help people to orient them self faster and find the right issue.

MAybe we could also ask them to comment on the issue if they start working on it to allow multiple people working the same issue?